metrics/integration.go

```diff
@@ -185,7 +176,6 @@ func init() {
 		IntegrationPipelineRunTotal,
 		SnapshotConcurrentTotal,
 		SnapshotDurationSeconds,
-		SnapshotInvalidTotal,
```

Comment:
Also definition on line 92 should be removed

Reply:
done